### PR TITLE
AZP: Organize yaml files 

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Fix failures like https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=24619&view=logs&jobId=607583ef-48a2-5666-f009-e509e61eab9f&j=607583ef-48a2-5666-f009-e509e61eab9f&t=ee1f994d-f2db-509a-3312-89f4055ae997:

```sh
+ export PROGRESS=0
+ PROGRESS=0
+ gcc --version
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

+ git clone --depth 1 -b v1.11.x https://github.com/openucx/ucx.git /scrap/azure/agent-08/AZP_WORKSPACE/1/s/ucx-1.11
fatal: destination path '/scrap/azure/agent-08/AZP_WORKSPACE/1/s/ucx-1.11' already exists and is not an empty directory.
##[error]Bash exited with code '128'.
```

### CODE REVIEW

**Comment by avildema** [comment]
may be move this test to separate yaml file? or in other PR?

**Comment by yosefe** [comment]
@avildema done

File: buildlib/pr/main.yml [hunk 1]
```diff
@@ -207,117 +100,7 @@ stages:
...
-          - checkout: self
-            clean: true
-            fetchDepth: 100
-
-          - bash: |
-              set -eE
-              . buildlib/tools/common.sh
-              prepare_build
-              ${WORKSPACE}/configure $(CONFIGURE_OPTS) --prefix=$(Build.Repository.LocalPath)/install
-              gcc -v
```
Comment (yosefe) [inline buildlib/pr/main.yml:235]: removed these in new file

File: buildlib/pr/distro.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+jobs:
+    # Perform test builds on relevant distributions.
+  - job: Distros
+    displayName: Build for distros
+    workspace:
+      clean: all
```
Comment (yosefe) [inline buildlib/pr/distro.yml:6]: added this to clean prev files

**Review by avildema — APPROVED** [review]
Good

**Pull request merged**